io-engine/src/bdev/nexus/nexus_io.rs

```diff
@@ -482,6 +489,12 @@ impl<'n> NexusBio<'n> {
                 })
         });
 
+        let mut reset_active = false;
+        if failed_device.is_some() {
+            let dev_name = failed_device.as_ref().unwrap();
+            reset_active = NvmeController::reset_active(dev_name);
+        }
```

Comment:
```suggestion
        let reset_active = failed_device.map(|dev_name| NvmeController::reset_active(dev_name)).unwrap_or_default();
```

Reply:
This code was added before I implement the nexus pause/resume. This is not needed now, will remove it